plivo/xml/ResponseElement.py

```diff
@@ -2,14 +2,39 @@
                        GetDigitsElement, HangupElement, MessageElement,
                        NumberElement, PlayElement, PlivoXMLElement,
                        PreAnswerElement, RecordElement, RedirectElement,
-                       SpeakElement, UserElement, WaitElement)
+                       SpeakElement, UserElement, WaitElement,
+                       BreakElement, EmphasisElement, LangElement
+                       )
 
 
 class ResponseElement(PlivoXMLElement):
     _name = 'Response'
     _nestable = [
-        'Conference', 'Dial', 'Number', 'User', 'DTMF', 'GetDigits', 'Hangup',
-        'Message', 'Play', 'PreAnswer', 'Record', 'Redirect', 'Speak', 'Wait'
+        'Conference',
+        'Dial',
+        'Number',
```

Comment:
Number, User, Break, Emphasis, Lang, P, S, W, Sub, Cont, SayAs, Prosody, Phoneme are not nestable in Response.